app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")

```

Comment:
what about extension?
Hint: .db extension is usually associated with SQLite vers2, while .db3 with SQLite 3.

Reply:
Thanks for advice! I didn't have any extension in pycharm or in a directory, so I decided that it should work well like this